src/beheer-header/beheer-header.html

```diff
@@ -13,11 +13,17 @@ <h4>is erfgoed</h4>
       </a>
       <div class="trianglePlaceholder"></div>
     </div>
+    <nav class="left menu-button">
```

Comment:
Het werkt op inventaris dus voor mij is het in orde. Nu gaan we daar wel altijd een nav met een lege ul hebben, misschien enkel die template met replaceable zetten en die nav met ul doorgeven in parent apps?